source/community/maintainers.rst

```diff
@@ -194,8 +194,24 @@ CI/CD Pipeline
 
 The Exegol project relies on a continuous integration and continuous deployment (CI/CD) pipeline for multiple scenarios. At the time of writing, Tue 31 Jan 2023, the pipeline is structured as follows:
 
-* the GitHub Actions platform is used on :doc:`the Exegol-images submodule </the-exegol-project/docker-images>`. Its workflows allow to build and push images on `the official Dockerhub registry <https://hub.docker.com/repository/docker/nwodtuhs/exegol>`_, run tests to make sure the tools are installed properly, run tests to help review pull requests, etc. GitHub Actions workflows are also being developped for packaging and publishing the Python wrapper on PyPI (`Exegol on PyPI <https://pypi.org/project/Exegol>`_).
-* no pipeline(s) yet on the Python wrapper, resources, docs, etc. But it's definitely in the roadmap.
+..  tabs::
+
+    ..  tab:: wrapper
+
+        The GitHub Actions platform is used on :doc:`the Exegol module </the-exegol-project/python-wrapper>`. Its workflows are used for internal and external pull requests, new releases and testing on every commit. The workflows build, and push Python packages on `the official PyPI registry <https://pypi.org/project/Exegol/>`_, and run tests to make sure everything works as it should.
+
+    ..  tab:: images
+
+        The GitHub Actions platform is used on :doc:`the Exegol-images submodule </the-exegol-project/docker-images>`. Its workflows allow to build and push images on `the official Dockerhub registry <https://hub.docker.com/repository/docker/nwodtuhs/exegol>`_, run tests to make sure the tools are installed properly, automatically export tools list to the documentaiton, run tests to help review pull requests, etc.
+
+        .. image:: /assets/gh_pipelines.png
+            :align: center
+            :alt: Pipelines (GitHub)
+
+    ..  tab:: docs
+
+        The GitHub Actions platform is used for the documentation you're reading. Its workflows are used to build on every commit to make sure everything works as it should, but also automatically merge changes between the various branches in order to help with development.
```

Comment:
It's the readthedocs pipeline that build the doc not GitHub action (copy paste fail).
But yeah for auto-merge we use GH action.

Reply:
It's building on every commit/PR for testing, adding more detail